Numpy/numpy_challenge.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+
+
+def arrays():
+    arr_1 = np.eye(3)
+    arr_2 = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]], dtype=float)
+    arr_3 = np.full((3, 3), 0)
+    return
+
+
+def matrix_multiplication(m1, m2):
+    return np.matmul(m1, m2)
+
+
+def multiplication_check(matrices):
+    for m in range(len(matrices) - 1):
+        if len(matrices[m][0]) != len(matrices[m + 1]):
+            return False
+    return True
+
+
+def multiply_matrices(matrices):
+    if multiplication_check(matrices) is False:
+        return
+    else:
+        res = matrices[0]
+        for m in matrices[1:]:
+            res = np.matmul(res, m)
+        return res
```

Comment:
Рабочий вариант, но можно воспользоваться функцией, которая сделает сразу всё
```suggestion
      return np.multidot(matrices)
```